contracts/lockdrop/src/raw_queries.rs

```diff
@@ -0,0 +1,37 @@
+use cosmwasm_std::{from_slice, Addr, Empty, QuerierWrapper, StdResult, Uint128};
+use cw_storage_plus::Path;
+use serde::{Deserialize, Serialize};
+
+/// @dev Returns generator deposit of tokens for the specified address
+pub fn raw_generator_deposit(
+    querier: QuerierWrapper,
+    generator: &Addr,
+    lp_token: &[u8],
+    address: &[u8],
+) -> StdResult<Uint128> {
+    #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Default)]
+    struct UserInfo {
```

Comment:
I think it's better to make this struct public in generator package and use it. We need this to track contract changes on the generators' side which are about to come (https://github.com/astroport-fi/astroport-core/pull/229).

Reply:
Done. Now UserInfo has only `amount` and can work with UserInfo and UserInfoV2. Thanks